lib/src/io/node.dart

```diff
@@ -80,6 +100,19 @@ _readFile(String path, [String encoding]) {
   }
 }
 
+String readStdin() async {
```

Comment:
This needs to be defined in `io/interface.dart` as well.

Also, it should be `Future<String>`.